lib/modules/neverEndingComments.js

```diff
@@ -0,0 +1,50 @@
+import * as _ from 'lodash';
+import { $ } from '../vendor';
+import { elementInViewport } from '../utils/dom';
+import * as SettingsConsole from './settingsConsole';
+
+export const module = {};
+
+module.moduleID = 'neverEndingComments';
+module.moduleName = 'Never Ending Comments';
+module.category = 'Comments';
+module.description = 'For those ultra-long comment pages, this will keep them flowing';
+module.options = {
+	loadChildComments: {
+		type: 'boolean',
+		value: false,
+		description: 'Whether or not on-screen \'load more comments\' links should be expanded when paused',
+	},
+};
+
+module.include = [
+	'comments',
+];
+
+module.go = () => {
+	window.addEventListener('scroll', _.debounce(handleScroll, 300));
+};
+
+function visibleLoaders() {
+	const loaders = module.options.loadChildComments.value ? $('span.morecomments a') : $('span.morecomments:last a');
+	return loaders.filter((i, el) => elementInViewport(el));
```

Comment:
Either:

Convert the jQuery object to an array and call .find on that:
```js
return loaders.toArray().find(...);
```

Use `document.querySelectorAll` instead of jQuery, convert that to an array, and call .find on that:
```js
const loaders = ... document.querySelectorAll('span.morecomments a') ...
return Array.from(loaders).find(...);
```

Use `document.querySelectorAll` instead of jQuery, and use its iterator (actually jQuery is iterable too):
```js
const loaders = ... document.querySelectorAll('span.morecomments a') ...
for (const el of loaders) {
    if (elementInViewport(el)) return el;
}
```